Sources/SwiftParser/Expressions.swift

```diff
@@ -2544,6 +2561,67 @@ extension Parser {
   }
 }
 
+// MARK: Platform Condition
+extension Parser {
+  mutating func parseCanImportExpression(_ canImportIdentifierExpr: RawIdentifierExprSyntax) -> RawExprSyntax {
+    let (unexpectedBeforeLeftParen, leftParen) = self.expect(.leftParen)
+
+    let (unexpectedBeforeImportPath, importPath) = self.expect(.identifier)
+
+    let comma = self.consume(if: .comma)
+    var versionTupleExpr: RawTupleExprElementSyntax?
+
+    if comma != nil {
+      versionTupleExpr = self.parseTupleExprElement(pattern: .none)
```

Comment:
Instead of parsing a `TupleExprSyntax` and then taking it apart again, I think it would be easier to just write the parsing code for `canImport` arguments themselves.

Reply:
Already refactored this part.😄